.github/workflows/live-preview.yaml

```diff
@@ -0,0 +1,75 @@
+name: Preview PR on galaxy.odahub.fr
+on:
+  pull_request:
```

Comment:
it should also run on change label @dsavchenko

Reply:
There is pull_request: labeled event there, isn't it enough?